Source/WebCore/loader/PingLoader.cpp

```diff
@@ -146,9 +146,15 @@ void PingLoader::sendPing(Frame& frame, const URL& pingURL, const URL& destinati
     FrameLoader::addHTTPOriginIfNeeded(request, SecurityPolicy::generateOriginHeader(document.referrerPolicy(), request.url(), sourceOrigin));
 
     frame.loader().updateRequestAndAddExtraFields(request, IsMainResource::No);
+
+    // https://www.w3.org/TR/2009/WD-html5-20090423/history.html#hyperlink-auditing
```

Comment:
Ugh, that's an old link. Could we please use this one instead?
https://html.spec.whatwg.org/multipage/links.html#hyperlink-auditing

Reply:
Absolutely.  The old one was strange with its referer.  Thanks